internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization.go

```diff
@@ -21,10 +21,10 @@ import (
 
 func ResourceAuthorization() *schema.Resource {
 	return &schema.Resource{
-		ReadContext:   resourceMongoDBAtlasCloudProviderAccessAuthorizationRead,
-		CreateContext: resourceMongoDBAtlasCloudProviderAccessAuthorizationCreate,
-		UpdateContext: resourceMongoDBAtlasCloudProviderAccessAuthorizationUpdate,
-		DeleteContext: resourceMongoDBAtlasCloudProviderAccessAuthorizationPlaceHolder,
+		ReadContext:   resourceCloudProviderAccessAuthorizationRead,
```

Comment:
In this package we have two resources, so to further shorten I would still need to leave `Authorization` or `Setup` in the method name, I can't simply use `resourceRead` because it would have a name conflict.

I would leave it as it is, can always be trivially renamed later if it's really needed.